notebooks/Integrate_NLP_Libraries.ipynb

```diff
@@ -2699,15 +2699,15 @@
        "66        [280, 283): 'Sir'          PROPN    sir   \n",
        "67        [284, 287): 'Not'            ADV    not   \n",
        "68          [287, 288): '-'          PUNCT   None   \n",
-       "69  [288, 297): 'Appearing'           NOUN   None   \n",
+       "69  [288, 297): 'Appearing'          PROPN   None   \n",
```

Comment:
seems wrong

Reply:
All the models I've seen have had trouble with "Sir Not-Appearing-In-This-Film". I'll chalk that up to a lack of subtle British humor in the training data ;)